packages/app/src/components/Live/DonationModal.tsx

```diff
@@ -32,18 +35,19 @@ export const DonationModal = (props: Props): ReactElement => {
     );
 
     const [number, onChangeNumber] = useState('');
-
     const usernameString = liveChallenge.creator?.username?.slice(0, 15);
     const [createDonation, { loading }] = useMutation<CreateDonationMutation>(
         CreateDonation
     );
 
     const handleCreateDonation = () => {
         const input = {
-            amount: utils.parseEther(number),
-            content: liveChallenge?.content,
+            amount: utils.parseEther(number).toString(),
+            content: props.content,
             challenge: liveChallenge?._id
         };
+        console.log('props.content: ', props.content);
+        console.log('utils.parseEther(number): ', utils.parseEther(number));
```

Comment:
@youngkidwarrior Are these consoles required? if not, I will remove it.

Reply:
Remove please thank you